github-actions/utils.ts

```diff
@@ -30,13 +30,16 @@ async function getJwtAuthedAppClient([appId, inputKey]: GithubAppMetadata) {
  * act on a repository, unlike the github-actions robot account which implicitly has access based on
  * where it was  executed from.
  */
-export async function getAuthTokenFor(app: GithubAppMetadata): Promise<string> {
+export async function getAuthTokenFor(
+  app: GithubAppMetadata,
+  useOrgInstallation: boolean = false,
+): Promise<string> {
   const github = await getJwtAuthedAppClient(app);
 
   const {id: installationId} = (
-    await github.apps.getRepoInstallation({
-      ...context.repo,
-    })
+    await (useOrgInstallation
```

Comment:
Wouldn't GitHub even with an org installation automatically be able to provide a token for the current context repo?

Reply:
See above comments.